daemon/libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -91,7 +91,7 @@ func enableBridgeNetFiltering(nfParam string) (retErr error) {
 func getKernelBoolParam(path string) (bool, error) {
 	line, err := os.ReadFile(path)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("reading '%s': %w", path, err)
```

Comment:
Curious; was information missing in the error, or did you pre-emptively add this? Because `os.ReadFile` (and other file operations from stdlib) _should_ usually return a `fs.PathError` / `os.PathError`, which already decorates the error with "Op" (e.g. `open` ), `Path`, and the error; https://github.com/golang/go/blob/768c51e368e05739854c1413fdf7fd129d01e482/src/io/fs/fs.go#L250-L255


```go
// PathError records an error and the operation and file path that caused it.
type PathError struct {
    Op   string
    Path string
    Err  error
}
```